### PR TITLE
Improve linter. Fix broken links, spelling errors, enable markdown table formatter

### PR DESCRIPTION
**Pull request by javydekoning**
*Issue #, if available:*
#491 

*Description of changes:*
- Fixed spelling errors found by linter
- Fixed markdown table formatting
- Fixed broken links in documentation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/admin-guide.md [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
-- [Tagging Policies](#tagging-policies)
-- [Pipelines](#pipelines)
-  - [Pipeline Parameters](#pipeline-parameters)
-  - [Chaining Pipelines](#chaining-pipelines)
-- [Integrating Slack](#integrating-slack)
-- [Check Current Version](#check-current-version)
-- [Updating Between Versions](#updating-between-versions)
-- [Removing ADF](#removing-adf)
-- [Troubleshooting](#troubleshooting)
+- [Administrator Guide](#administrator-guide)
```
Comment (sbkok) [inline docs/admin-guide.md:3]: Can you rerun this table to include the latest sections added by the PRs that were open alongside this one?

**Review by sbkok — CHANGES_REQUESTED** [review]
Great improvement, thank you for adding this!

File: docs/admin-guide.md [hunk 2]
```diff
@@ -123,11 +127,11 @@ The Master account *(also known as root)* is the owner of the AWS Organization.
...
+The Deployment Account is the gatekeeper for all deployments throughout an Organization. Once the baselines have been applied to your accounts via the bootstrapping process, the Deployment account connects the dots by taking source code and resources from a repository *(Github / CodeCommit / S3)* and into the numerous target accounts and regions as defined in the deployment map files via AWS CodePipeline. The Deployment account holds the [deployment_map.yml](#adfconfig) file(s) which defines where, what and how your resources will go from their source to their destination. In an Organization there should only be a single Deployment account. This is to promote transparency throughout an organization and to reduce duplication of code and resources. With a single Deployment Account teams can see the status of other teams deployments while still being restricted to making changes to the deployment map files via [Pull Requests](https://docs.aws.amazon.com/codecommit/latest/userguide/pull-requests.html).
```
[the remark this answers is not in the document]
Comment (javydekoning) [reply docs/admin-guide.md]: 👍 done

File: docs/admin-guide.md (hunk 1 above)
Comment (javydekoning) [reply docs/admin-guide.md:3]: 👍 done

**Review by sbkok — APPROVED** [review]

**Review by StewartW — APPROVED** [review]

**Pull request merged**